src/assets/less/site/cheatsheet.less

```diff
@@ -0,0 +1,9 @@
+.cheatsheet-icon {
+  .make-sm-column(6);
```

Comment:
As mentioned in `cheatsheet.html`, I'd love to have this new less partial only have to contain the print-specific styling for the cheatsheet. Can you try the following?

1) removing the non-print rules here and re-instating the `col-md-4 col-sm-6 col-lg-3` classes in the `cheatsheet.html` view.
2) keep the print-specific rule in this file and confirm that it is obeyed when the cheatsheet is printed.